src/utils/authenticate-with-code.ts

```diff
@@ -31,7 +32,8 @@ export async function authenticateWithCode(
   if (response.ok) {
     const data = (await response.json()) as AuthenticationResponseRaw;
     return deserializeAuthenticationResponse(data);
-  } else {
-    console.log("error", await response.json());
   }
+
+  const error = (await response.json()) as any;
```

Comment:
Nit: `json` already resolves to `any`. 🙈 

```suggestion
    const error = await response.json();
```